docs/requirements/Detailed-list-of-all-possible-dnscheck-messages.md

```diff
@@ -0,0 +1,1330 @@
+# All DNSCheck Messages
+
+The list is in alphabetical order.
+
+##Address module
```

Comment:
Would it be possible to fix the Markdown syntax for the headings?

Reply:
@mattias-p, it is beyond the original scope of this PR but I'll do it.